tests/src/main/java/com/hazelcast/stabilizer/tests/synthetic/SyntheticTest.java

```diff
@@ -239,8 +243,7 @@ private long getBackupDelayNanos() {
                 return 0;
             }
 
-            long d = Math.abs(random.nextLong());
-            return d % backupDelayNanos;
```

Comment:
Before someone asks: FindBugs finding due to a problem with `Math.abs()` and `Long.MIN_VALUE`